assets/src/edit-story/components/tablist/panel.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+/**
+ * Internal dependencies
+ */
+import { Icons } from '../../../design-system';
+import { PANEL_STATES } from './constants';
+import {
+  Badge,
+  ButtonText,
+  IconContainer,
+  PanelWrapper,
+  SmallHeadline,
+  TabButton,
+  TabPanel,
+} from './styles';
+
+const Panel = ({
+  badgeCount = 0,
+  children,
+  className,
+  isExpanded,
+  onClick,
+  status = PANEL_STATES.NORMAL,
+  title,
+}) => {
+  const panelId = useMemo(uuidv4, []);
+
+  return (
+    <PanelWrapper className={className} isExpanded={isExpanded}>
+      <TabButton
+        aria-controls={panelId}
+        aria-selected={isExpanded}
+        onClick={onClick}
+        role="tab"
+        status={status}
+      >
+        <ButtonText>
+          <IconContainer>
+            <Icons.ChevronDownSmall />
+          </IconContainer>
+          <SmallHeadline id={`${title}-${panelId}`}>{title}</SmallHeadline>
+        </ButtonText>
+        <Badge>
+          <SmallHeadline>{badgeCount}</SmallHeadline>
```

Comment:
By default this is setting as an `h1` headline.  Instead of a headline, which I know design used for visuals, let's use a standard `<Text size={...PRESETPATH.BLAHBLAH.SMALL} isBold>`  

This will eliminate the need to set the line-height for it at all.

Reply:
>![image](https://user-images.githubusercontent.com/22185279/123321258-f9e8fc00-d4ef-11eb-99e9-66ea46cc84e5.png)

love this